docs/docs/cloud/how-tos/copy_threads.md

```diff
@@ -0,0 +1,165 @@
+# Copying Threads
+
+You may wish to copy an existing thread in order to keep the existing history and experiment with different runs in the future.
```

Comment:
Maybe something like "You may wish to copy (i.e. "fork") an existing thread in order to keep the existing thread's history and create independent runs that do not affect the original thread. This guide shows how you can do that."